cmake/MujocoOptions.cmake

```diff
@@ -79,6 +79,30 @@ else()
 endif()
 
 option(MUJOCO_BUILD_MACOS_FRAMEWORKS "Build libraries as macOS Frameworks" OFF)
+option(MUJOCO_INSTALL_PLUGINS "Install MuJoCo plugins" ON)
```

Comment:
Do you mind making exactly the same changes to SimulateOptions.cmake?

SimulateOptions.cmake is used for SIMULATE_STANDALONE here: https://github.com/google-deepmind/mujoco/blob/91192393a0293d97bbb4bdd0fe7a16d8faf43bc0/simulate/CMakeLists.txt#L60

and the two files should be kept in sync.

Reply:
Done and rebased. 

Just an OT curiosity, if the two files must be kept in sync, there is any reason why we can't use the same `MujocoOptions.cmake` directly in `mujoco/simulate/CMakeLists.txt`, for example using `include(../cmake/MujocoOptions.cmake)` instead of `include(SimulateOptions)` ? Indeed inclusion of a file in a sibling directory is a bit ugly, but perhaps it would be easier to manage then two files that need to be manually kept in sync (unless you have something on the Google side to keep them in sync, in that case probably the problem is just for external contributors).

Reply:
Internally they are just one file that gets split in our CI so ya, this is a problem we save specifically for external contributors 🙂

Reply:
Annoyingly it looks like copybara still refuses to ingest this 😠 

@nimrod-gileadi should I just copy-paste this into a CL?
Alternatively we could implement @traversaro's proposal to do the file duplication/referencing in cmake rather than copybara?

I'll approve yet again, maybe it'll work... 🤷‍♂️